test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -0,0 +1,35 @@
+(ns puppetlabs.services.legacy-routes.legacy-routes-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.trapperkeeper.core :as tk]
+            [puppetlabs.trapperkeeper.app :as tka]
+            [puppetlabs.trapperkeeper.testutils.logging :as logging]
+            [puppetlabs.http.client.sync :as http-client]
+            [puppetlabs.kitchensink.core :as ks]
+            [puppetlabs.services.legacy-routes.legacy-routes-service :as legacy-routes-service]
+            [puppetlabs.services.master.master-service :as master-service]
+            [schema.test :as schema-test]
+            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
+            [me.raynes.fs :as fs]))
+
+(def test-resources-dir
+  "./dev-resources/puppetlabs/services/legacy_routes/legacy_routes_test")
+
+(use-fixtures
+  :once
+  schema-test/validate-schemas
+  (jruby-testutils/with-puppet-conf (fs/file test-resources-dir "puppet.conf")))
+
+(defn http-get [path]
+  (http-client/get
+    (str "https://localhost:8140" path)
+    bootstrap/request-options))
+
+(deftest ^:integration legacy-routes
```

Comment:
Per the conversation in SERVER-526, I had been expecting that we'd try to have at least some level of coverage for validating that "auth.conf" rules created per the v3 routes would be evaluated against requests made to the corresponding legacy routes.  If that's going to cause a significant amount of rework -- like more than 1/2 hour -- I'm open to putting that in a follow-up ticket in the interest of getting this merged sooner.